javascript-custom-methods/calculator.js

```diff
@@ -0,0 +1,32 @@
+/* exported calculator */
+var calculator = {
+  add: function(x,y) {
+    return x + y;
+  },
+  subtract: function (x, y) {
+    return x -y;
+  },
+  multiply: function (x, y) {
+    return x * y;
+  },
+  divide: function (x, y) {
+    return x / y;
+  },
+  square: function (x, y) {
+    return x ** 2;
+  },
+  sumAll: function(numbers) {
+    var start = 0;
+    for (i = 0; i < numbers.length; i++) {
+      start += numbers[i];
+    } return start;
```

Comment:
I recommend moving your `return` statements to a new line, it's really hard to read when it's like this. Most developers will just skim your code going line by line, and it'll be easy to miss this when trying to review code quickly.